include/RAJA/policy/cuda/reduce.hpp

```diff
@@ -604,22 +604,23 @@ struct Reduce_Data {
   bool own_device_ptr;
 
   Reduce_Data()
-  {
-    reset(T(), T());
-  }
+    : Reduce_Data(T(),T()){};
 
   /*! \brief create from a default value and offload information
    *
    *  allocates PinnedTally to hold device values
    */
-  Reduce_Data(T initValue, T identity_)
+  explicit Reduce_Data(T initValue, T identity_)
```

Comment:
Two arg ctors do not need to be declared explicit to avoid unwanted implicit type conversion. They would only need this if an arg had a default value.